### DISCUSSION TITLE
Image recognition over indexed photos

### DISCUSSION THREAD

**Discussion by psyciknz** [photoprism/photoprism#2813] [Q&A]
How can I run Image recognition again. If I assume the models are more advanced, how would I rerun the image recognition for categories?

Also as a follow on, if I wanted to manually set a category label,  say Cat to a bunch of photos, is the old option to manually add it to each image?  It does seem it apply the single entry to all selected photos.

**Comment by graciousgrey** [comment]
There is no command to only run image classification. You can trigger a complete rescan for this (it's also possible for single directories).

[Batch edit](https://github.com/photoprism/photoprism/issues/271) is not implemented yet, so yes at the moment you need to add the label each photo

**Reply by psyciknz** [reply]
Thanks for the update on the batch edit.   

Regarding a reindex.   If I've manually added labels etc, will they be retained?  Or are they cleared?

**Reply by lastzero** [reply]
Labels will only be added, they won't get removed. Especially when set manually.
